en/middleware/micrortps.md

```diff
@@ -45,18 +50,22 @@ The main elements of the architecture are the client and agent processes shown i
   In a typical configuration they will both be on the same system (e.g. a development computer, Linux companion computer or compute board), connected to the *Client* over a Wifi link or via USB.
 
 
-## Architectural overview for a ROS2/ROS application pipeline
+### ROS2/ROS application pipeline
+
+The application pipeline for ROS2 is very straightforward! 
+Because ROS2 uses DDS/RTPS as its native communications middleware, you can create a ROS2 listener or advertiser node to publish and subscribe to uORB data on PX4, via the *PX4 Fast RTPS Bridge*. This is shown below.
+
+> **Note** You do need to make sure that the message types, headers and source files used on both client and agent side (and consequently, on the ROS nodes) are generated from the same Interface Description Language (IDL) files. The `px4_ros_com` package provides some of the needed infrastructure for generating messages and headers needed by ROS2.
```

Comment:
@TSC21 This needs review in particular. Ie confirm that this is what px4_ros_com does